### PR TITLE
Added my profile

### PR DESCRIPTION
**Pull request by Akshat-Mishra101**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [] My change requires changes to the documentation.
- [] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1694"></a>

### CODE REVIEW

**Review by maharshi-sinha — APPROVED** [review]

**Review by vedanshi555 — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/Akshat-Mishra101.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://twitter.com/@0xakshat",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@_akshat",
+      "icon": "medium"
+    },
+    {
+      "name": "Follow me on Linkedin",
```
Comment (Panquesito7) [inline public/data/Akshat-Mishra101.json:23]: I recommend capitalizing the `i` here. What do you think about this? 🙂
```suggestion
      "name": "Follow me on LinkedIn",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Comment (Akshat-Mishra101) [reply public/data/Akshat-Mishra101.json:23]: Sure, let me fix this right away :)

**Comment by Akshat-Mishra101** [comment]
Hey @Panquesito7, I've made the changes :)

**Comment by Akshat-Mishra101** [comment]
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 👌 This code looks good

Thanks @vedanshi555 , I'm new to open source :)

**Comment by vedanshi555** [comment]
> > __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 👌 This code looks good
> 
> Thanks @vedanshi555 , I'm new to open source :)

great!! thank you for contributing! welcome to this community, glad to have you here😁

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks for contributing!! 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**